src/components/macroview/topArea/TriggerModal.tsx

```diff
@@ -137,24 +137,32 @@ export default function TriggerModal({ isOpen, onClose }: Props) {
                 gap="4px"
                 minH="42px"
                 bg={secondBg}
-                alignItems="center"
-                rounded="md"
+                justifyContent="center"
+                rounded={borderRadiusStandard}
                 p="9px"
                 shadow="inner"
               >
                 {items.length === 0 && (
-                  <Text>
-                    Set up to 4 keys* or a mouse button to use as the trigger.
+                  <Text textAlign="center">
+                    {'Set up to 4 keys or a mouse button to use as the trigger'}
                   </Text>
                 )}
                 {items.map((element, index) => (
-                  <Kbd variant="brand" h="fit-content" key={element}>
+                  <Kbd
+                    fontSize="md"
+                    variant="brand"
+                    h="fit-content"
+                    key={element}
+                  >
                     {displayNames[index]}
                   </Kbd>
                 ))}
               </Flex>
               <HStack w="full" justifyContent="space-between">
-                <Text fontSize="xs">* (1 non-modifer, up to 3 modifiers)</Text>
+                <VStack alignItems="left">
+                  <Text fontSize="sm">{`1x non-modifier, up to 3x modifiers in any order.`}</Text>
```

Comment:
Clarifying text, changes to styling

Reply:
Fixed: [1cbb2eb](https://github.com/WootingKb/wooting-macros/pull/194/commits/1cbb2eb81395844e6711a75ae1c62665308a3e07)